blazor/media-query/media-query-reusable.md

```diff
@@ -0,0 +1,201 @@
+---
+layout: post
+title: Reuse of Blazor Media Query Component | Syncfusion
+description: Checkout and learn here all about how to use the Media Query component at the global level and re-use on all pages and much more.
+platform: Blazor
+control: Media Query
+documentation: ug
+---
+
+# Global level and Reuse of Blazor Media Query Component
+
+You can reuse the `Media Query` component across different parts of your application, which allows you to maintain a consistent and adaptive design. By reusing the Blazor Media Query component, you can maintain a consistent responsive design throughout your application.
+
+In the below example, the `Media Query` component defined in `ReusableMediaQuery.razor` file and used this tag in the `index.razor` file by changing built-in breakpoints as per requirement.
```

Comment:
Description changes,

In the below example, the `Media Query` component is defined in the `ReusableMediaQuery.razor` file and the `<ReusableMediaQuery>` tag is used in the `index.razor` file by changing built-in breakpoints as per requirement.